src/nvim/regexp.c

```diff
@@ -3768,9 +3771,11 @@ static long bl_maxval;
  * Returns FALSE when there is no match.  Leaves reginput and reglnum in an
  * undefined state!
  */
-static int 
-regmatch (
-    char_u *scan              /* Current node. */
+static int
+regmatch(
+    char_u *scan,               // Current node.
+    proftime_T *tm,
+    int     *timed_out
```

Comment:
original patch added documentation (comments) to these parameters. Why not include that?

I will fix this in the merge-commit.